editoast/fga/src/lib.rs

```diff
@@ -483,11 +483,15 @@ pub mod test_utilities {
         use client::DEFAULT_OPENFGA_MAX_TUPLES_PER_WRITE;
         use client::Limits;
 
-        let address = std::env::var("OPENFGA_HOST").unwrap_or_else(|_| "localhost".to_string());
-        let port = std::env::var("OPENFGA_PORT")
-            .unwrap_or_else(|_| "8091".to_string())
-            .parse()
-            .expect("invalid port");
+        let fga_api_url = url::Url::parse(
+            &std::env::var("FGA_API_URL").unwrap_or_else(|_| "http://localhost:8091".to_string()),
+        )
+        .expect("invalid openfga api url");
```

Comment:
```suggestion
        .expect("invalid FGA_API_URL");
```

better provide the env var in case it fails to prevent too much confusion